docs/reference/transform/apis/put-transform.asciidoc

```diff
@@ -24,6 +24,12 @@ Requires the following privileges:
 * source indices: `read`, `view_index_metadata`
 * destination index: `read`, `create_index`, `index`. If a `retention_policy` is configured, the `delete` privilege is
```

Comment:
You could also modify the start transform docs page to remove the index permission requirements if they're no longer needed.